src/data/faq.json

```diff
@@ -417,6 +417,15 @@
                             "The digital proof of vaccination is generated in the doctor's practice or in a vaccination center. After entering or transferring the data, a vaccination certificate token (QR code) will be created for you, which you can scan directly with the Corona-Warn-App (CWA) or can be handed to you on a paper printout and later scanned with the CWA.  You can then manage your digital vaccination certificates on your smartphone."
                         ]
                     },
+                    {
+                        "title": "What is a booster vaccination and how does it appear in the CWA?",
+                        "anchor": "vac_booster_basics",
+                        "active": true,
+                        "textblock": [
+                            "<div class='container'><div class='row'><div class='col-sm'><a href='/assets/img/faq/vac_booster_basics_en.png' target='_blank'><img src='/assets/img/faq/vac_booster_basics_en.png' alt='Certificate List' width='300'></a></div><div class='col-sm'><p>A booster vaccination (revaccination) is a vaccination process designed to refresh waning vaccination protection after basic immunization. In most cases, but not always, it takes place according to the recommendations of the authorities after a period of about 5-6 months after the last vaccination procedure. (Example: completion of the vaccination series for basic immunization on 08 Jul 2021, possible booster vaccination on: 08 Jan 2022)</p><p>Each booster vaccination can be attested by a digital COVID certificate, which you can scan into the Corona-Warn-App. Important: please ensure that the personal data (surname, first name and date of birth) are shown to match the existing vaccination certificates. Only then will it be possible to assign the new certificate to the booster vaccination of the basic immunization. (Example: same academic title or indication of the full double name).</p><p>Booster shots provide immediate vaccine protection.</p><p>Based on the previous vaccination series, the booster certificate will be documented '3 of 3' (in case of previous basic vaccination with 2 doses) or with '2 of 2' (as vaccination optimization according to the Janssen agent from Johnson & Johnson). In the latter case, however, the CovPassCheck app, Corona-Warn-App and CovPass app cannot display this correctly in all cases for technical reasons.</p><p>The '1 of 1' label for a booster vaccination indicates that it is the first and only vaccination after a COVID-19 disease recovery. Vaccinations with Johnson & Johnson are an exception.</p><p> See also: <a href='#vac_booster_jj' target='blank'>Why does a 14-day wait for full vaccine protection after a booster vaccination appear in the Corona-Warn-App?</a></p></div></div></div>",

```

Comment:
1. Change "basic vaccination" to "basic immunization":

```suggestion
                            "<div class='container'><div class='row'><div class='col-sm'><a href='/assets/img/faq/vac_booster_basics_en.png' target='_blank'><img src='/assets/img/faq/vac_booster_basics_en.png' alt='Certificate List' width='300'></a></div><div class='col-sm'><p>A booster vaccination (revaccination) is a vaccination process designed to refresh waning vaccination protection after basic immunization. In most cases, but not always, it takes place according to the recommendations of the authorities after a period of about 5-6 months after the last vaccination procedure. (Example: completion of the vaccination series for basic immunization on 08 Jul 2021, possible booster vaccination on: 08 Jan 2022)</p><p>Each booster vaccination can be attested by a digital COVID certificate, which you can scan into the Corona-Warn-App. Important: please ensure that the personal data (surname, first name and date of birth) are shown to match the existing vaccination certificates. Only then will it be possible to assign the new certificate to the booster vaccination of the basic immunization. (Example: same academic title or indication of the full double name).</p><p>Booster shots provide immediate vaccine protection.</p><p>Based on the previous vaccination series, the booster certificate will be documented '3 of 3' (in case of previous basic immunization with 2 doses) or with '2 of 2' (as vaccination optimization according to the Janssen agent from Johnson & Johnson). In the latter case, however, the CovPassCheck app, Corona-Warn-App and CovPass app cannot display this correctly in all cases for technical reasons.</p><p>The '1 of 1' label for a booster vaccination indicates that it is the first and only vaccination after a COVID-19 disease recovery. Vaccinations with Johnson & Johnson are an exception.</p><p> See also: <a href='#vac_booster_jj' target='blank'>Why does a 14-day wait for full vaccine protection after a booster vaccination appear in the Corona-Warn-App?</a></p></div></div></div>",
```

---

2. 
> Vaccinations with Johnson & Johnson are an exception.

What is meant here? A booster with J?J is an exception? If yes, the linked FAQ entry does not provide the user more information on this.

Reply:
@Ein-Tim  basic immunization, fine! Thanks. Regarding 

> What is meant here? A booster with J&J is an exception? If yes, the linked FAQ entry does not provide the user more information on this.

We are updating this article as well.